### PR TITLE
Add the ability to soak rags and cotton balls on disinfectant.

### PR DESCRIPTION
**Pull request by Zhid13**
Due to the rather harsh avaliability of disinfectant to early game survivors, this patch lets you soak 4 rags with one charge of disinfectant, with a 20% less probabilities to cure the infected wound. It adds the same to cotton balls, with only 2 of them being soaked, and just 15% less chances.

### CODE REVIEW

**Comment by Treah** [comment]
Why do you keep closing and opening this PR? The previous one you had was #23278

**Comment by Zhid13** [comment]
Sorry, first I accidentaly closed it. I deleted the before one due to commit saturation. This one won't be closed and reopened again without sense.

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -1288,5 +1288,42 @@
...
+            "head_power" : 1,
+            "torso_power" : 1,
+            "limb_power" : 1,
+            "bite" : 0.8,
+            "move_cost" : 300
+        }
     }
+
+
+
```
Comment (Cyrano7) [inline data/json/items/comestibles/med.json]: Looks like you have some extra spaces in here that could be trimmed down.

**Comment by Night-Pryanik** [comment]
I don't know. I'd rather not add another direct-heal item when we plan (at least I hope so) to change the healing mechanics so there should be no direct-heal items - https://github.com/CleverRaven/Cataclysm-DDA/issues/16017#issuecomment-204938566.

File: data/json/recipes/other/medical.json [hunk 2]
```diff
@@ -111,5 +111,35 @@
...
     ]
-  }
+  },
+  {
+    "type": "recipe",
+    "result": "disinrag",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MEDICAL",
+    "skill_used": "firstaid",
+    "skills_required": [ "firstaid", 1 ],
```
Comment (Coolthulhu) [inline data/json/recipes/other/medical.json]: Skills required and skill used should not overlap. Use one or the other, or make one of them a different skill.

**Comment by illi-kun** [comment]
@Zhid13 Your nickname can be estimated as quite offensive, I'm not sure about policy of github and the current repo, could you explain the reason of using it?

**Comment by Zhid13** [comment]
The reason of the name isn't related to the russian word meant there, but to a latinoamerican slang for "cool" often used in memes. "Chido" "Zhido" "Zhid"

**Comment by Voker57** [comment]
> @Zhid13 Your nickname can be estimated as quite offensive, I'm not sure about policy of github and the current repo, could you explain the reason of using it?
> 
> Explanation for non-russian readers: 'Zhid' means '[Yid](https://en.wikipedia.org/wiki/Yid)'.

Your remark can be estimated as highly russian-centric, Zhid is a legitimate word in many languages. Please refrain from from making offtopic remarks in this repo's issues and consider removing "kun" from your nick as it can be estimated as an racial slur https://en.wikipedia.org/wiki/Coon#Racially_charged_usage

**Pull request merged**